--- origsrc/ImageMagick-6.8.8-5/coders/emf.c	2014-01-23 15:35:58.000000000 -0600
+++ src/ImageMagick-6.8.8-5/coders/emf.c	2014-02-11 01:49:54.556580700 -0600
@@ -42,7 +42,7 @@
 #include "magick/studio.h"
 #if defined(MAGICKCORE_WINGDI32_DELEGATE)
 #  if !defined(_MSC_VER)
-#    if defined(__CYGWIN__)
+#    if defined(__CYGWIN__) || defined(__MINGW32__) || defined(__MINGW64__)
 #      include <windows.h>
 #    else
 #      include <wingdi.h>
